tools/lock_inspect/evm_connection.rb

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env ruby
+
+require 'pg'
+require 'yaml'
+require 'optparse'
+
+def error_exit(e, conn, error_message = '', exit_code = 1)
+  puts e.message
+  puts error_message unless error_message.empty?
+  conn.close if conn
+  exit(exit_code)
+end
+
+def process_app_name(app_name)
+  if app_name.end_with?('..')
+    raise "The application name for MIQ worker: {#app_name} is truncated"
```

Comment:
Sure. I think we can raise it here, but change on its capture. so it can do 1)  just put the error message and abort or 2) put error msg and ignore this line and continue processing it. Do 1) or 2) can do based on a option given to it. For this specific process_app_name, maybe it's more general to just raise error?